source/viz.md

```diff
@@ -1169,41 +1262,36 @@ Bar plot of size for Earth's largest 12 landmasses.
 :::
 
 
-The plot in {numref}`islands_bar_top` is definitely clearer now,
-and allows us to answer our question
-("Which are the top 7 largest landmasses continents?") in the affirmative.
-But the question could be made clearer from the plot
-by organizing the bars not by alphabetical order
-but by size, and to color them based on whether they are a continent.
-The data for this is stored in the `landmass_type` column.
-To use this to color the bars,
-we use the `color` argument to color the bars according to the `landmass_type`
+The plot in {numref}`islands_bar_top` is clearer now,
+and allows us to answer our question:
+"Which are the top 7 largest landmasses continents?".
```

Comment:
this isn't the question -- the question should be 

"Are the top 7 largest landmasses continents?" 

which is why you need "in the affirmative" after.

Reply:
Hmm this is what we have in the question further up:

![image](https://github.com/UBC-DSCI/introduction-to-datascience-python/assets/4560057/3df4aaee-e727-4361-9ac4-f50333e7dbe4)

Which to me is more similar to "Which of the top 7 largest landmasses are continents?" (fixing a couple of typos)

Reply:
We should keep the wording the same throughout if possible -- so "are the continents the largest 7" or the like

(also if the original Q were just "which of the top 7 are continents" I'd just plot 7 instead of whatever we're doing now, 12 or something like that)

Reply:
I guess it makes it more complicated that our original question is two-fold: both "Are the continents (North / South America, Africa, Europe, Asia, Australia, Antarctica) Earth’s seven largest landmasses?" and then "If so, what are the next few largest landmasses after those?". I updated to include both of those with the same phrasing as in the original question to keep it consistent, let me know what you think.

I believe the reason we have 12 instead of 7 is to show of the color encoding which would just contain one color if we only included the top 7 since those are all continents:

![image](https://github.com/UBC-DSCI/introduction-to-datascience-python/assets/4560057/96efec68-7134-4470-bf95-41c0444821da)